### DISCUSSION TITLE
update service catalogs

### DISCUSSION THREAD

**Discussion by moos3** [gruntwork-io/knowledge-base#441] [Help] [answered]
Is there or where is the documentation for updating the reference arch with the latest version of the service catalog and amis? I have been digging around the docs I haven't found something that is clear. I tried to do it via just updating the version of the end of the strings and I get running in detected head in the ci/cd pipeline. So looking for some guidance. 

---

<ins datetime="2022-05-31T16:51:59Z">
  <p><a href="https://support.gruntwork.io/hc/requests/108683">Tracked in ticket #108683</a></p>
</ins>

**Comment by eak12913** [comment]
We've got [this](https://docs.gruntwork.io/guides/reference-architecture/update-to-service-catalog/) guide. Is this what you were looking for?

**Reply by moos3** [reply]
Doesn't really answer the question if your refarch customer since feburary of this year. Theres nothing to migrate from the old system to the new service-catalog as we are already on it.

**Comment by yorinasub17** [comment, marked as answer]
The best way to upgrade your Reference Architecture over time is to follow the release notes in the Service Catalog for each module, and update each component one at a time as new versions of the Service Catalog are released.

If there are major architectural changes, we would publish a specific guide such as the one @eak12913 mentioned above. Otherwise, most changes are only to the underlying module components, which are captured in the release notes.
